cmd/destroy/destroy.go

```diff
@@ -240,16 +244,21 @@ func (dc *destroyCommand) runDestroy(ctx context.Context, opts *Options) error {
 	}
 
 	oktetoLog.AddToBuffer(oktetoLog.InfoLevel, "Destroying...")
+
 	data := &pipeline.CfgData{
 		Name:      opts.Name,
 		Namespace: namespace,
 		Status:    pipeline.DestroyingStatus,
 		Filename:  opts.ManifestPathFlag,
 	}
-	cfg, err := pipeline.TranslateConfigMapAndDeploy(ctx, data, c)
-	if err != nil {
-		return err
+	var cfg *v1.ConfigMap
+	if !utils.LoadBoolean(model.OktetoWithinDeployCommandContextEnvVar) {
```

Comment:
what does this `OktetoWithinDeployCommandContextEnvVar`mean?

Reply:
It means that the command is being executed by an `okteto deploy`, all commands executed by `okteto deploy` have this environment variable